packages/input-edit/src/components/InputEdit/InputEdit.js

```diff
@@ -39,50 +51,57 @@ const InputEdit = ({entityKey, initializeTable, initializeSearch, initializeInfo
       .filter(child => child.componentType === 'action')
       .map(child => ({...child, scope: 'detail'}))
     : [], [inputDataForm])
-  const selection = actions.getSingleEntitySelection('Input', entityKey)
 
-  return <StyledSplitPane
-    defaultSize={300}
-    minSize={250}
-    resizerStyle={resizerStyle}
-    split="vertical">
-    <StyledSplitPanelWrapper key={'sidebar'}>
-      <StyledSplitPane defaultSize={300}
-        minSize={300}
+  if (valid === false) {
+    notify('error', 'client.component.input-edit.error.title', message)
+    return null
+  }
+  return <LoadMask required={[valid || message]}>
+    {valid
+      ? <StyledSplitPane
+        defaultSize={300}
+        minSize={250}
         resizerStyle={resizerStyle}
-        split="horizontal">
-        <StyledSplitPanelWrapper key="search">
-          <InputEditSearch/>
+        split="vertical">
+        <StyledSplitPanelWrapper key={'sidebar'}>
+          <StyledSplitPane defaultSize={300}
+            minSize={300}
+            resizerStyle={resizerStyle}
+            split="horizontal">
+            <StyledSplitPanelWrapper key="search">
+              <InputEditSearch/>
+            </StyledSplitPanelWrapper>
+            <StyledSplitPanelWrapper key="information">
+              <InputEditInformation/>
+            </StyledSplitPanelWrapper>
+          </StyledSplitPane>
         </StyledSplitPanelWrapper>
-        <StyledSplitPanelWrapper key="information">
-          <StyledGutter>
-            <Icon icon="horizontal-rule"/>
-          </StyledGutter>
-          <InputEditInformation/>
+        <StyledSplitPanelWrapper key="table">
+          {actionDefinitions.map(definition =>
+            <actions.Action
+              key={definition.id}
+              definition={definition}
+              selection={selection}/>
+          )}
+          <InputEditTable/>
+          <InputEditPagination/>
         </StyledSplitPanelWrapper>
       </StyledSplitPane>
-    </StyledSplitPanelWrapper>
-    <StyledSplitPanelWrapper key="table">
-      {actionDefinitions.map(definition =>
-        <actions.Action
-          key={definition.id}
-          definition={definition}
-          selection={selection}/>
-      )}
-      <InputEditTable/>
-      <InputEditPagination/>
-    </StyledSplitPanelWrapper>
-  </StyledSplitPane>
+      : null}
+  </LoadMask>
 }
 
 InputEdit.propTypes = {
-  entityKey: PropTypes.string.isRequired,
+  selection: PropTypes.object.isRequired,
+  validation: PropTypes.object.isRequired,
```

Comment:
Maybe define shape for `validation`? I personally would use `PropType.object` only if you don't know how the given object is structured and if you really don't care (e.g. because you only pass it down to the child components). In my experience any developer that doesn't know a certain component is able to understand and use it much quicker if the prop types (especially the more complex ones) are defined properly.

The `valid` property of this prop seems to be required (as far as i've seen the rendering of the component doesn't fail but there won't be rendered a whole lot if there is also no `message`).